enhancements/update/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,184 @@
+---
+title: cincinnati-graph-data-package-distribute-and-consume-in-a-container-image-repository
+authors:
+  - "@steveeJ"
+reviewers:
+  - @openshift/openshift-team-cincinnati-maintainers
+  - @openshift/openshift-team-cincinnati
+  - @openshift/team-cincinnati-operator
+approvers:
+  - TBD
+creation-date: 2020-05-07
+last-updated: 2020-05-07
+status: provision
+see-also:
+-
+replaces: []
+superseded-by: []
+---
+
+# cincinnati-graph-data: package, distribute, and consume in a container image repository
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+### Content trust
+Trust in the container image registry and in the container image content is possibly an issue if we an approach where Cincinnati uses a mutable tag, e.g. `latest` tag to receive the most recent content from the container registry.
+Assuming we're using Quay as an image registry, [which does not support Docker Content Trust](https://support.coreos.com/hc/en-us/articles/115000289114-Docker-Content-Trust-support).
+If we decide that to ensure the image integrity, which means do we have which do not further complicate the workflow?
```

Comment:
Need to reword this paragraph. Possibly a typo or two but the point or question is not clear.

Reply:
I am also not sure what is being communicated in the last sentence.

Reply:
Thanks, I changed the phrase. The first occurrence of "that" was superfluous.